README.md

```diff
@@ -257,6 +259,36 @@ To then limit access to a specific set of users or groups, you can follow the st
 
 By default, the deployed Azure web app allows users to chat with all your indexed data. You can enable an optional login system using Azure Active Directory to restrict access to indexed data based on the logged in user. Enable the optional login and document level access control system by following [this guide](./LoginAndAclSetup.md).
 
+### Restricting network traffic
+
+By default, this application will allow connections from any IP address. The backend resources like Cognitive Search and Open AI are protected by API security. The web application does not have any authentication mechanism unless [it is explicitly configured](#enabling-authentication) and will allow any IP address to connect by default.
+To restrict access of the web application, the search endpoint, Azure Open AI, Azure Storage and the Cognitive Service Form Recognizer to an IP or network address, set the `ALLOWED_IP` environment variable.
+
+For example, to allow only the IP address `43.133.5.124` to connect:
+
+```console
+azd env set ALLOWED_IP 43.133.5.124
+```
+
+You can also use CIDR notation to specify an IP address range, for example to allow the IP addresses 43.133.5.0-43.133.5.255 (the /24 network)
+
+```console
+azd env set ALLOWED_IP 43.133.5.0/24
+```
+
+Please note that the IP address configured will need to include the one you run `azd up` from, since deployment connects to the web application and the `prepdocs.py` script connects to the Cognitive Search index, Storage API, and Form Recognizer APIs.
+
+### Enabling private endpoints
+
```

Comment:
Given the difficulties I had with just using the IP restriction without the Vnet, I propose merging IP section into this one, so that we can at least have vnet support. As far as I can tell, vnet deploy works.